src/pudl_archiver/orchestrator.py

```diff
@@ -127,46 +131,35 @@ def __init__(
         self.depositor = ZenodoDepositor(upload_key, publish_key, session, self.sandbox)
         self.downloader = downloader
 
-        # TODO (daz): don't hold references to the depositions at the instance level.
-        self.deposition: Deposition | None = None
-        self.new_deposition: Deposition | None = None
-        self.deposition_files: dict[str, DepositionFile] = {}
-
         self.dry_run = dry_run
 
         self.create_new = create_new
-        self.deposition_settings = deposition_settings
+        self.dataset_settings_path = deposition_settings
```

Comment:
Should we just change the `deposition_settings` parameter to `dataset_settings_path`?

Reply:
Yes! Good thought!